lib/compilers/ppci.js

```diff
@@ -1,3 +1,26 @@
+// Copyright (c) 2012-2018, Matt Godbolt
```

Comment:
You can put your name here, it's your code ;)

Reply:
Hehe, okay. I note you too for the regex trickz.